src/include/function/boolean/vector_boolean_operations.h

```diff
@@ -6,7 +6,39 @@
 namespace kuzu {
 namespace function {
 
-class VectorBooleanOperations : public VectorOperations {
+template<typename FUNC>
```

Comment:
Can u try to put these back into `VectorBooleanOperations`. This probably won't cause compilation problem in MSVC.